src/revChatGPT/ChatGPT.py

```diff
@@ -102,8 +108,8 @@ def ask(self, prompt, conversation_id=None, parent_id=None):
                     "content": {"content_type": "text", "parts": [prompt]},
                 },
             ],
-            "conversation_id": conversation_id or self.conversation_id,
-            "parent_message_id": parent_id or self.parent_id or str(uuid.uuid4()),
+            "conversation_id": conversation_id,
+            "parent_message_id": parent_id,
```

Comment:
This could cause problems if there is no existing parent_id. It has to be random on the first call

Reply:
Re-added random UUID on first call